src/index.ts

```diff
@@ -136,14 +355,43 @@ const dmMachine = createMachine(
       "speak.greeting": ({ context }) => {
         context.spstRef.send({
           type: "SPEAK",
-          value: { utterance: "Hello world!" },
+          value: { utterance: "Hello, welcome to your air-conditioner!" },
         });
       },
-      "speak.how-can-I-help": ({ context }) =>
+        "speak.Feel": ({ context }) =>
         context.spstRef.send({
           type: "SPEAK",
-          value: { utterance: "How can I help you?" },
+          value: { utterance: "How do you feel today, do you want to turn the air-conditionor on? If yes, on which mode would you like it , on which intensity, and in which direction?" },
```

Comment:
The first question should be more open-ended. Like "how do you feel today?". Then you should be fill the slots (on/off, intensity, direction, temperature (?) ) with specific questions for each of the slots. In order to implement this, you will need the guards that operate on the value of RECOGNISED event. Now I don't see that you have implemented any guards.